src/main/kotlin/com/hibob/academy/dao/OwnerDao.kt

```diff
@@ -0,0 +1,41 @@
+package com.hibob.academy.dao
+
+import org.jooq.DSLContext
+import org.jooq.RecordMapper
+import org.jooq.Record
+import java.util.*
+
+
+class OwnerDao(private val sql: DSLContext) {
+
+    private val ownerTable = OwnersTable.instance
+
+    private val ownerMapper = RecordMapper<Record, Owner> { record ->
+        Owner(record[ownerTable.companyId], record[ownerTable.employeeId], record[ownerTable.name])
+    }
+
+    fun getOwners(): List<Owner> =
+        sql.select(ownerTable.companyId, ownerTable.employeeId, ownerTable.name)
+            .from(ownerTable)
+            .fetch(ownerMapper)
+
```

Comment:
You should get as parameter companyId and fetch by companyId

Reply:
should I do it in all the gettres?
getPetsByOwnerId
getOwnerByPetId